Makefile

```diff
@@ -17,12 +17,12 @@
 all: flutter
 
 # available backends
-WITH_TFLITE?=1
+WITH_TFLITE?=0
 WITH_QTI?=0
 WITH_SAMSUNG?=0
 WITH_PIXEL?=0
 WITH_MEDIATEK?=0
-WITH_APPLE?=0
+WITH_APPLE?=1
```

Comment:
Please do not change the default values in this Makefile.
The Windows and Android tests failed because it does not support the Apple backend.
Anyway, I will update the build script later to use the TFLite backend explicitly.

Reply:
Thanks, got it!